src/aio/ops.zig

```diff
@@ -44,6 +44,22 @@ pub const Fsync = struct {
     userdata: usize = 0,
 };
 
+/// std.io.poll
+pub const Poll = struct {
```

Comment:
Windows can't support this operation, it should return error in queuCallback and the handler in start should lead to unreachable

Reply:
Perhaps it could even error comptime by setting this struct to panic on windows :thinking: